src/patch_parse.c

```diff
@@ -578,6 +587,13 @@ static int parse_hunk_body(
 			old_lineno = -1;
 			break;
 
+		case '\\':
+			prefix = 0;
```

Comment:
@pks-t I'm not quite sure what the prefix `int` is supposed to do, and I kind of find it odd that it is subtracted from `parse_ctx.line` which is a `char *`. What is it?

Reply:
Nevermind... simple pointer arithmetic 🙄